OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -147,17 +473,240 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <ab type="ruling">Ruling and pricking are not visible.</ab>
                                 </layout>
                             </layoutDesc>
                             
                         </objectDesc>
                         
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <date notBefore="1550" notAfter="1799"/>
+                           <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">The title and numeral at the beginning of each canticle; elements of punctuation signs.</seg>
+                                <desc>Fine and regular handwriting. The letters are slightly slanted towards the right, broad spaces between characters. ፮ and ጵ have
+                                    the modern form (<locus target="#23ra #36vb"/>).</desc>
+                            </handNote>
+                            
+                            <handNote xml:id="h2" script="Ethiopic" corresp="#a1">
+                                <locus from="1r" to="2v"/>
+                                <date notBefore="1800" notAfter="2006"/>
+                                <seg type="ink">Black.</seg>
+                            </handNote>
+                            
+                            <handNote xml:id="h3" script="Ethiopic" corresp="#a2">
+                                <locus from="3r" to="4r"/>
+                                <date notBefore="1800" notAfter="2006"/>
+                                <seg type="ink">Black.</seg>
+                            </handNote>
+                            
+                            <handNote xml:id="h4" script="Ethiopic" corresp="#a3">
+                                <locus target="#4v"/>
+                                <date notBefore="1800" notAfter="1974"/>
+                                <seg type="ink">Black.</seg>
+                                <desc>An untrained hand.</desc>
+                            </handNote>
+                            
+                            <handNote xml:id="h5" script="Ethiopic" corresp="#a6">
+                                <locus from="47r" to="51v"/>
+                                <date notBefore="1800" notAfter="2006"/>
+                                <seg type="ink">Black.</seg>
+                                <desc>An untrained, irregular hand.</desc>
                             </handNote>
                         </handDesc>
                         
+                        <additions>
+                            <list>
+                                <item xml:id="a1" corresp="#h2">
+                                    <locus from="1r" to="2v"/>
+                                    <desc type="MagicText">Magic prayer containing excerpts from the Psalms of David. The name or names mentioned at the end of the text have
```

Comment:
I see the pros and cons of both terms and have no personal opinion :) Would everyone prefer "protective" @SusanneHummel @Gnisci @ElianaDalSasso ?

Reply:
I have no objection to either solution. I do know that some frown upon "magic" nowadays.

Reply:
It is far from my expertise but in Coptic literature there is the same problem. The term "magic" is not totally correct but it is deeply rooted in literature, therefore it is still used. I think that, if "protective" works we could opt for it.

Reply:
I've added this to the meeting agenda so that it will not be forgotten, but maybe we can solve this here, there is already a tendency towards "protective". Let's wait for @SusanneHummel

Reply:
I am afraid that "magic", intended as linked to supernatural or wondrous forces or phenomena, might lead us (or the reader) on a slippery ground of anthropological questions (how do we distinguish between "devotional" and "magic" activities? And what about "magic" and "medical" behaviours in context where the practice aims to heal somebody?). "Magic" has been a common word in our field in the past decades, but is also a bit problematic and probably the "magic literature" still needs further research. Our catalogue simply is not the right place to deal with this complex question, so that's why I would call it quits by using a more cautious term as "protective".

Reply:
I also would prefer to use "protective" instead of "magic".